Framework/Beamline/inc/MantidBeamline/ComponentInfo.h

```diff
@@ -53,7 +53,9 @@ class MANTID_BEAMLINE_DLL ComponentInfo {
       m_componentRanges;
   boost::shared_ptr<const std::vector<size_t>> m_parentIndices;
   Mantid::Kernel::cow_ptr<std::vector<Eigen::Vector3d>> m_positions;
-  Mantid::Kernel::cow_ptr<std::vector<Eigen::Quaterniond>> m_rotations;
+  Mantid::Kernel::cow_ptr<std::vector<
+      Eigen::Quaterniond, Eigen::aligned_allocator<Eigen::Quaterniond>>>
```

Comment:
After including StdVector -> includes src/StlSupport/StdVector specialisation, I think you will automatically get the right allocator. i.e. no need to express here.

Reply:
The Eigen documentation says that one needs to **both** the `StdVector` header an the aligned allocator. 

https://eigen.tuxfamily.org/dox/group__TopicStlContainers.html#title2

I tried using the `EIGEN_DEFINE_STL_VECTOR_SPECIALIZATION` that tries to hide the second requirement. Ensuring they are both defined everywhere it is necessary and not defined twice was more trouble than it is worth.